openrtb_ext/request_wrapper.go

```diff
@@ -200,7 +200,7 @@ func (rw *RequestWrapper) RebuildRequest() error {
 	return nil
 }
 
-func (rw *RequestWrapper) rebuildImp() error {
+func (rw *RequestWrapper) RebuildImp() error {
```

Comment:
This is intended to be a helper method of the higher level rebuild request.

Reply:
We wanted to build only impression object and to avoid checking for other objects we have exported to build only required object